lib/src/content/wolt_modal_sheet_animated_switcher.dart

```diff
@@ -129,45 +123,38 @@ class _WoltModalSheetAnimatedSwitcherState
 
   void _resetScrollPositions() {
     _scrollPositions.clear();
-    _scrollPositions = [
-      for (int i = 0; i < _pagesCount; i++) ValueNotifier(0.0)
-    ];
+    _scrollPositions = [for (int i = 0; i < _pagesCount; i++) ValueNotifier(0.0)];
   }
 
   void _resetScrollControllers() {
     _scrollControllers.clear();
     _scrollControllers = [
       for (int i = 0; i < _pagesCount; i++)
-        (_page.scrollController ??
-            ScrollController(initialScrollOffset: _scrollPositions[i].value))
+        (_page.scrollController ?? ScrollController(initialScrollOffset: _scrollPositions[i].value))
     ];
   }
 
   void _subscribeToCurrentPageScrollPositionChanges() {
     for (final scrollController in _scrollControllers) {
       scrollController.addListener(() {
         if (_currentPageScrollController.hasClients) {
-          _currentPageScrollPosition.value =
-              _currentPageScrollController.position.pixels;
+          _currentPageScrollPosition.value = _currentPageScrollController.position.pixels;
         }
       });
     }
   }
 
   void _subscribeToSoftKeyboardClosedEvent() {
     _softKeyboardVisibilitySubscription =
-        KeyboardVisibilityController().onChange.listen((
-      bool visible,
-    ) async {
-      if (!visible) {
-        /// Wait for closing soft keyboard animation to finish before emitting new value.
-        await Future.delayed(const Duration(milliseconds: 250));
-        final int lastEventId = _softKeyboardClosedNotifier.value.eventId;
-        final newEventId = lastEventId + 1;
-        _softKeyboardClosedNotifier.value =
-            SoftKeyboardClosedEvent(eventId: newEventId);
-      }
-    });
+        KeyboardVisibilityController().onChange.listen((bool visible,) async {
+          if (!visible) {
+            /// Wait for closing soft keyboard animation to finish before emitting new value.
+            await Future.delayed(const Duration(milliseconds: 250));
```

Comment:
250ms should be a const either in top of the file or somewhere where `defaultWoltModalTransitionAnimationDuration` is also located.

Reply:
done in here: 3a2364af7d963f66d0d39575076a96c6e07f140c